app/assets/javascripts/alchemy/alchemy.dialog.js.coffee

```diff
@@ -105,9 +104,6 @@ class window.Alchemy.Dialog
   # Initializes the Dialog body
   init: ->
     Alchemy.GUI.init(@dialog_body)
-    Alchemy.Tinymce.initWith
```

Comment:
Is there a way with custom elements to get back this behavior? Admittedly it's a rarely used feature, but we would at least tell people how to upgrade.

Reply:
It is possible, but only on the component itself:

<img width="484" alt="Aufnahme 2023-08-28 at 22 51 39@2x" src="https://github.com/AlchemyCMS/alchemy_cms/assets/122262394/ba7f8977-7c7b-4647-9830-a720cb7250f1">

<img width="382" alt="Aufnahme 2023-08-28 at 22 51 46@2x" src="https://github.com/AlchemyCMS/alchemy_cms/assets/122262394/a18b2e15-9b88-486b-80d3-5a4c2fa65af1">